subsys/usb/host/usbh_core.c

```diff
@@ -50,44 +50,52 @@ static void dev_connected_handler(struct usbh_context *const ctx,
 				  const struct uhc_event *const event)
```

Comment:
If this function needs to be used outside, by the HUB, then it is possible to remove the `static`, give it an API name such as `usbh_device_connected()`, then add it to `usbh_device.h`, as an example.